app.js

```diff
@@ -1,11 +1,96 @@
 /* Imports */
-
+import { renderVillain } from './utils.js';
 /* Get DOM Elements */
+const villainsEl = document.getElementById('villains');
+const playerHPEl = document.getElementById('player-hp');
+const playerImgEl = document.getElementById('player-img');
+const inputEl = document.getElementById('villain-input');
+const buttonEl = document.getElementById('villain-button');
+const defeatedCountEl = document.getElementById('defeated-count');
 
 /* State */
+let defeatedCount = 0;
+let playerHp = 10;
 
+const villains = [
+    {
+        name: 'Bowser',
+        hp: 5,
+    },
+    {
+        name: 'Goomba',
+        hp: 3,
+    },
+    {
+        name: 'Boo',
+        hp: 2,
+    },
+];
 /* Events */
+buttonEl.addEventListener('click', () => {
+    const villainName = inputEl.value;
+
+    if (!villainName) {
+        return;
+    }
+
+    const newVillain = {
+        name: villainName,
```

Comment:
here you can do name: `villainName || 'villain Math.random()'` to generate a random villain name as the rubric says.